redstone/client.py

```diff
@@ -460,6 +468,21 @@ def unwrap(self, key_id, ciphertext, aad=None):
         resp = self._action(key_id, "unwrap", data)
         return base64.b64decode(resp['plaintext'].encode())
 
+    def createTKEK(self, expiration, maxAllowedRetrievals):
+        data = {'expiration': expiration,
+                'maxAllowedRetrievals': maxAllowedRetrievals}
+
+        resp = self.session.post("%s/api/v2/import_token" % self.endpoint_url,
+                                 json=data)
+
+        self._validate_resp(resp)
+        return
+
+    def getTKEK(self):
```

Comment:
same as above, getTKEK -> get_transport_key/get_import_token

also, we don't need an ID here? https://cloud.ibm.com/apidocs/key-protect#retrieve-a-transport-key-by-id

Reply:
we don't. we've moved to using the bluemix instance id as the key to the import_token table in the DB.